PowerEditor/installer/nativeLang/swedish.xml

```diff
@@ -343,49 +343,49 @@
 				<Item id="1"    name="Sök nästa"/>
 				<Item id="1722" name="Sök föregående"/>
 				<Item id="2"    name="Stäng"/>
-				<Item id="1620" name="Sök efter:"/>
+				<Item id="1620" name="Sök &amp;efter:"/>
 				<Item id="1603" name="Matcha &amp;hela ord"/>
 				<Item id="1604" name="&amp;Matcha små/STORA bokstäver"/>
 				<Item id="1605" name="&amp;Reguljärt uttryck"/>
 				<Item id="1606" name="&amp;Loopa"/>
-				<Item id="1614" name="Antal sökträffar"/>
+				<Item id="1614" name="&amp;Antal sökträffar"/>
 				<Item id="1615" name="Hitta alla"/>
-				<Item id="1616" name="Bokmärk rad"/>
+				<Item id="1616" name="&amp;Bokmärk rad"/>
 				<Item id="1618" name="Rensa tidigare markering vid varje sökning"/>
-				<Item id="1611" name="Ersätt med:"/>
+				<Item id="1611" name="Ers&amp;ätt med:"/>
 				<Item id="1608" name="&amp;Ersätt"/>
 				<Item id="1609" name="Ersätt &amp;alla"/>
 				<Item id="1687" name="Vid förlorat fokus"/>
 				<Item id="1688" name="Alltid"/>
 				<Item id="1632" name="I mar&amp;kering"/>
 				<Item id="1633" name="Rensa alla markeringar"/>
-				<Item id="1635" name="Ersätt allt i alla öppna dokument"/>
-				<Item id="1636" name="Sök allt i alla öppna dokument"/>
-				<Item id="1654" name="Filter:"/>
-				<Item id="1655" name="Mapp:"/>
+				<Item id="1635" name="Ersätt allt i alla &amp;öppna dokument"/>
+				<Item id="1636" name="Sök allt i alla ö&amp;ppna dokument"/>
```

Comment:
While you're at it, take a look at the Sök-dialog and consider whether "Sök allt" shouldn't be "Sök alla" :-)
I was just about to submit a PR and noticed you were already working in this file, so if you like, you can fix this.